mono/mini/debugger-state-machine.h

```diff
@@ -0,0 +1,71 @@
+/**
+ * \file
+ * Types for the debugger state machine and wire protocol
+ *
+ * Author:
+ *   Alexander Kyte (alkyte@microsoft.com)
+ *
+ * (C) 2018 Microsoft, Inc.
+ *
+ */
+#ifndef __MONO_DEBUGGER_STATE_MACHINE__
+#define __MONO_DEBUGGER_STATE_MACHINE__
+
+
+#include <config.h>
```

Comment:
You shouldn't include config.h in a header.

Reply:
I missed this one.

Reply:
Fixed, thanks